packages/frontend/src/pages/Settings.styles.tsx

```diff
@@ -20,28 +20,13 @@ export const Title = styled(H5)`
     tex-align: left;
 `;
 
-export const ExpandableWrapper = styled.div`
-    box-shadow: none;
-    padding: 0 5px 0 2px;
-    margin-top: 12px;
-`;
-
-export const SettingsItems = styled.div`
-    padding-left: 27px;
-`;
-export const CollapseTrigger = styled(Button)`
-    font-weight: 600;
-    padding-left: 0;
-    width: 100%;
+export const SidebarMenu = styled(Menu)``;
```

Comment:
we don't need this empty styled component

Reply:
removed 👍